### PR TITLE
feat(tutorial): complete step 3

### PR DESCRIPTION
**Pull request by fpasqua1980**
Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: src/content/RepoPage/RepoPage.js [hunk 1]
```diff
@@ -1,5 +1,72 @@
+import { Link, DataTableSkeleton, Pagination } from 'carbon-components-react';
 import React from 'react';
 import RepoTable from './RepoTable';
+import { gql } from 'apollo-boost';
+import { Query } from 'react-apollo';
+import React, { useState } from 'react';
+
+const [totalItems, setTotalItems] = useState(0);
```
Comment (kellychurchill) [inline src/content/RepoPage/RepoPage.js]: These 3 constants should be declared inside the `RepoPage` const.

File: src/content/RepoPage/RepoPage.js [hunk 2]
```diff
@@ -70,4 +137,33 @@ const RepoPage = () => {
   );
 };
 
+<Query query={REPO_QUERY}>
```
Comment (kellychurchill) [inline src/content/RepoPage/RepoPage.js]: This should replace the `<RepoTable headers={headers} rows={rows} />` line.

File: src/content/RepoPage/RepoPage.js (hunk 1 above)
Comment (fpasqua1980) [reply src/content/RepoPage/RepoPage.js]: Changed Done.

File: src/content/RepoPage/RepoPage.js (hunk 2 above)
Comment (fpasqua1980) [reply src/content/RepoPage/RepoPage.js]: Changed Done

**Comment by kellychurchill** [comment]
@fpasqua1980 Your Netlify preview build is still failing. Please ensure that the project runs locally before committing your updates.

**Comment by fpasqua1980** [comment]
> Your Netlify preview build failed. Please see comments and ensure that the project runs locally before committing your updates.

I'm gotting the following error message in the browser and not sure how to figure that out.

TypeError: _ref2.data is undefined
RepoPage/<
src/content/RepoPage/RepoPage.js:102

   99 | <div className="bx--row repo-page__r1">
  100 |   <div className="bx--col-lg-16">
  101 |     <Query query={REPO_QUERY}>
> 102 |         {({ loading, error, data: { organization } }) => {
      | ^  103 |         // Wait for the request to complete
  104 |         if (loading)
  105 |           return (

**Comment by kellychurchill** [comment]
@fpasqua1980 Take another look at the tutorial step 3 https://www.carbondesignsystem.com/tutorial/react-step-3

Your `RepoPage.js` file is not correct.

**Comment by fpasqua1980** [comment]
> @fpasqua1980 Take another look at the tutorial step 3 https://www.carbondesignsystem.com/tutorial/react-step-3
> 
> Your `RepoPage.js` file is not correct.

Now its working. Thank you.

File: src/content/RepoPage/RepoPage.js [hunk 3]
```diff
@@ -28,45 +65,80 @@ const headers = [
...
+  }));
 
 const RepoPage = () => {
+  const [totalItems, setTotalItems] = useState(0);
+  const [firstRowIndex, setFirstRowIndex] = useState(0);
+  const [currentPageSize, setCurrentPageSize] = useState(10);
   return (
-    <div className="bx--grid bx--grid--full-width bx--grid--no-gutter repo-page">
-      <div className="bx--row repo-page__r1">
-        <div className="bx--col-lg-16">
```
Comment (kellychurchill) [inline src/content/RepoPage/RepoPage.js]: These 3 `<div>` wrappers should not be removed.

**Comment by kellychurchill** [comment]
👍 Looks good - 🎉 You are one step closer to completion of the Carbon React tutorial and earning the digital credential.

**Pull request closed without merging**